x86/bin/install_script.sh

```diff
@@ -107,8 +108,12 @@ if [ "$dl_path" = "" ] ; then
 	echo "SDK could not be set up. Relocate script unable to find ld-linux.so. Abort!"
 	exit 1
 fi
-#executable_files="'"$($SUDO_EXEC find "$native_sysroot" -type f -perm +111)"'"
-executable_files=$($SUDO_EXEC find "$native_sysroot" -type f -perm +111 -exec printf "\"%s\" " {} \; )
+
+for opt in $FIND_OPTS
+do
+    executable_files=`$SUDO_EXEC find "$native_sysroot" -type f -perm "$opt" -exec printf "\"%s\" " {} \;`
```

Comment:
**"Within the backquoted style of command substitution, <backslash> shall retain its literal meaning"**
I need that behaviour for the \; that marks the end of the -exec part.

I couldn't make it work with $(), anyway. Maybe you can.